build/ios/mojom/mojom_objc_generator.py

```diff
@@ -409,13 +428,15 @@ def is_move_only_kind(kind):
             return typestring
         return "const %s&" % typestring
 
-    def _GetObjCPropertyModifiers(self, kind):
-        modifiers = ['nonatomic']
+    def _GetObjCPropertyModifiers(self, kind, inside_union=False):
+        modifiers = {'nonatomic'}
         if (mojom.IsArrayKind(kind) or mojom.IsStringKind(kind) or
                 mojom.IsMapKind(kind) or mojom.IsStructKind(kind)):
-            modifiers.append('copy')
+            modifiers.add('copy')
+            if inside_union:
+                modifiers.add('nullable')
```

Comment:
Realizing I need to add more times where `nullable` is added (nested union for instance)